requirements.txt

```diff
@@ -37,8 +37,11 @@ SpeechRecognition==3.1.3
 telepot==4.1
 
 # Text To Speech Module
-pygame==1.9.2b6
+pygame>=1.9.2b6
 pyvona==1.1
 
 # Creating Sphinx Language Model
-poster==0.8.1
\ No newline at end of file
+poster==0.8.1
+
+# Weather
```

Comment:
>Maybe include that pywapi dependency just in the requirements-dev.txt for now

this is maybe better solution will try it next.

#14 is caused because the `requirements.txt` contain only `pywapi==0.3.8`. if it run directly with `pip install -r requirements.txt` it will search the package only in pypi.

note for #14 : use commit a7b0838d9ac80f320577b7f7759828102ea156ec , fix commit 6c3efde8357231865681ff40f605ba3d71702a0d

#7 the first and second test is kinda similar with #14, but the third test done by @asdofindia can find the url for pywapi package only by adding `--allow-unverified pywapi`. i have tried to create this third test by creating the text file contain only `pywapi==0.3.8` and run the similar command  

    (temp_py2) q@q-N-a:~/.git/Melissa-Core$ pip install -r temp_req.txt --allow-external pywapi --allow-unverified pywapi
    DEPRECATION: --allow-external has been deprecated and will be removed in the future. Due to changes in the repository protocol, it no longer has any effect.
    DEPRECATION: --allow-unverified has been deprecated and will be removed in the future. Due to changes in the repository protocol, it no longer has any effect.
    Collecting pywapi==0.3.8 (from -r temp_req.txt (line 1))
      Could not find a version that satisfies the requirement pywapi==0.3.8 (from -r temp_req.txt (line 1)) (from versions: )
    No matching distribution found for pywapi==0.3.8 (from -r temp_req.txt (line 1))

pip version 

    pip 8.1.2 from /home/q/.virtualenvs/temp_py2/local/lib/python2.7/site-packages (python 2.7)

note for #7 : use commit 2fc8950bbc18f7ddff7830e054f0e346a1e189a4 , fix commit 07690c1e9c2e79432d34a81ae7e30fa23433a080

i would love to ask @asdofindia and @neilnelson to try testing using this commit in new virtualenv and check if everything is working.

in the meantime i will do the first suggestion and move it to reqs-dev text.